lib/mrsk/configuration.rb

```diff
@@ -61,7 +60,6 @@ def traefik_hosts
     roles.select(&:running_traefik?).flat_map(&:hosts)
   end
 
-
```

Comment:
Extra spaces are intentional to separate sections of related methods.

Reply:
alright